doc/getting_started/getting_started.rst

```diff
@@ -20,22 +20,73 @@ Follow one of the following guides for your host operating system.
    macOS <installation_mac.rst>
    Windows <installation_win.rst>
 
-Clone the Zephyr Repository
-***************************
+.. _get_the_code:
+
+Get the source code
+*******************
+
+Zephyr's multi-purpose :ref:`west` tool lets you easily get the Zephyr project
+source code, instead of manually cloning the Zephyr repos along with west
+itself.
 
-To clone the Zephyr source code repository from GitHub:
+Bootstrap west
+==============
+
+Install the bootstrapper for Zephyr's command-line tool, :ref:`west` in a
+shell or ``cmd.exe`` prompt:
 
 .. code-block:: console
 
-   git clone https://github.com/zephyrproject-rtos/zephyr
+   # Linux
+   pip3 install --user west
+
+   # macOS and Windows
+   pip3 install west
+
+.. note::
+   See :ref:`gs_python_deps` for additional clarfication on using the
+   `--user` switch.
+
+Additional information about west's structure can be found
+:ref:`in the relevant west documentation section <west-struct>`.
+
+Clone the Zephyr Repositories
+=============================
+
+Clone the Zephyr source code repositories from GitHub using the ``west`` tool:
+
+.. code-block:: console
+
+   west init zephyrproject
```

Comment:
What is the process for changing the default manifest in the manifest repository? Are there PRs and are those tested thru shippable also?

Reply:
There are PRs but they are not tested by shippable yet.

Reply:
Then more weight to the argument of not just fetching the master tip here.

Reply:
right now west is not pulling the zephyr tree, because this was already fetched by shippable and the following lines deal with pointing git to the code in the PR, so we have exactly the same as we have things today without west. How to deal with multi-repo changes and manifest versions is not subject of this PR AFAIK.